wandb/sdk/data_types/trace_tree.py

```diff
@@ -431,6 +431,8 @@ def log(self, name: str) -> None:
             name: The name of the trace to be logged
         """
         trace_tree = WBTraceTree(self._span, self._model_dict)
+        # NOTE: Does not work for reinit="create_new" runs.
```

Comment:
i believe all of this file needs to be removed as it a precursor to weave

Reply:
Can you elaborate? Is this something we can already do in a future PR, or is there some refactoring we have to complete first?

Reply:
we wanted to do it for a while but it was blocked by weave team, we can re-raise it with growth + weave let me do it now.